Description: Fix babel configuration
Author: Nicolas Mora <babelouest@debian.org>
Forwarded: not-needed
--- a/.babelrc
+++ b/.babelrc
@@ -2,15 +2,9 @@
   env: {
     test: {
       presets: [
-        [ 'env', {
+        [ '@babel/env', {
           loose: true,
-          targets: { node: 4 },
-          exclude: [
-            // skip some almost-compliant features on Node.js v4.x
-            'transform-es2015-block-scoping',
-            'transform-es2015-classes',
-            'transform-es2015-for-of',
-          ]
+          targets: { node: 4 }
         } ]
       ],
       plugins: [
@@ -19,14 +13,9 @@
     },
     coverage: {
       presets: [
-        [ 'env', {
+        [ '@babel/env', {
           loose: true,
-          targets: { node: 4 },
-          exclude: [
-            'transform-es2015-block-scoping',
-            'transform-es2015-classes',
-            'transform-es2015-for-of'
-          ]
+          targets: { node: 4 }
         } ]
       ],
       plugins: [
@@ -36,14 +25,9 @@
     },
     rollup: {
       presets: [
-        [ 'env', {
+        [ '@babel/env', {
           loose: true,
           targets: { node: 4 },
-          exclude: [
-            'transform-es2015-block-scoping',
-            'transform-es2015-classes',
-            'transform-es2015-for-of'
-          ],
           modules: false
         } ]
       ]
