upstream git has babel removed

Forwarded: not-required

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -1,5 +1,5 @@
 import isBuiltin from 'is-builtin-module';
-import babel from 'rollup-plugin-babel';
+import babel from '@rollup/plugin-babel';
 import tweakDefault from './build/rollup-plugin';
 
 process.env.BABEL_ENV = 'rollup';
@@ -13,7 +13,7 @@
   ],
   plugins: [
     babel({
-      runtimeHelpers: true
+      babelHelpers: 'bundled'
     }),
     tweakDefault()
   ],
